lib/entities/quote/DutchQuote.ts

```diff
@@ -275,12 +279,65 @@ export class DutchQuote implements IQuote {
     return this.amountInStart;
   }
 
+  // The number of seconds from now that order decay should begin
+  public get startTimeBufferSecs(): number {
+    if (this.request.config.startTimeBufferSecs !== undefined) {
+      return this.request.config.startTimeBufferSecs;
+    }
+
+    if (this.isOpenQuote()) {
+      return OPEN_QUOTE_START_TIME_BUFFER_SECS;
+    }
+
+    return DEFAULT_START_TIME_BUFFER_SECS;
+  }
+
+  // The number of seconds from startTime that decay should end
+  public get auctionPeriodSecs(): number {
+    if (this.request.config.auctionPeriodSecs !== undefined) {
+      return this.request.config.auctionPeriodSecs;
+    }
+
+    switch (this.chainId) {
+      case 1:
+        return 60;
```

Comment:
nit: magic number

Reply:
yeaaa since these are like configs by chain tho the alternative is like a constants mapping? i guess thats fine

Reply:
ugg then cant do default easily though, tbh i think this is simpler and more expressive in this case